tests/sanity/test/apiml/test-caching-service-apis-via-gateway.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2018, 2019
+ */
+
+const expect = require('chai').expect;
+let request;
+let key = 'testKey';
+let value = 'testValue';
+let authenticationCookie;
+
+const testUtils = require('./utils');
+const CACHING_PATH = '/cachingservice/api/v1/cache';
+
+let assertStatusCodeCreated = (response) => {
+  expect(response.status).to.equal(201);
+};
+
+let assertStatusCodeOk = (response) => {
+  expect(response.status).to.equal(200);
+  expect(response.data).to.not.be.empty;
+};
+
+let assertStatusNoContent = (response) => {
+  expect(response.status).to.equal(204);
+};
+
+let getToken = async () => {
+  const uuid = testUtils.uuid();
+  testUtils.log(uuid, 'ZOWE_CACHING_SERVICE_START value: ' + process.env.ZOWE_CACHING_SERVICE_START);
+  return await testUtils.login(uuid);
+};
+
+describe('test caching service via gateway', function() {
```

Comment:
A potentially neater way, rather than having an if is to use an conditional skip eg:
```
const cachingServiceEnabled = process.env.ZOWE_CACHING_SERVICE_START !== undefined && process.env.ZOWE_CACHING_SERVICE_START == 'true'
(cachingServiceEnabled ? describe : describe.skip)('test caching service via gateway', function() {
```
?

Reply:
it works, thanks! I will replace it